server/routes/admin/approval/index.js

```diff
@@ -21,17 +20,17 @@ const adminApproval = async (req, res) => {
 };
 
 const _approveOrRejectNewUser = async (req, res) => {
-  const username = _parseEscapedInputAndSanitizeUsername(req.body.username);
-
   try {
-    const login = await _findLoginMatchingUsername(username);
+    const username = _parseEscapedInputAndSanitizeUsername(req.body.username);
+
+    const login = await models.login.findByUsername(username);
 
-    // Make sure we have the matching user
     if (login && login.id && username === login.username) {
-      const workplace = await _findWorkplace(login.user.establishment.id);
-      const user = await _findUser(login.user.id);
+      const workplace = await models.establishment.findbyId(login.user.establishmentId);
+      const user = await models.user.findByLoginId(login.user.id);
+
+      var workplaceIsUnique = await _workplaceIsUnique(workplace.uid, req.body.nmdsId);
```

Comment:
Can this be changed for a `const`?
```suggestion
      const workplaceIsUnique = await _workplaceIsUnique(workplace.uid, req.body.nmdsId);
```